service/src/test/java/bio/terra/workspace/common/utils/TestUtils.java

```diff
@@ -0,0 +1,11 @@
+package bio.terra.workspace.common.utils;
+
+import java.util.Random;
+
+public class TestUtils {
+  private static final Random RANDOM = new Random();
+
+  public static String appendRandomNumber(String string) {
+    return string + "-" + RANDOM.nextInt(100000);
```

Comment:
there is a uniqueName helper method in ControlledResourceFixtures btw. (I've been using it) Can we just move that one here?